content/learn/workflow.adoc

```diff
@@ -11,29 +11,31 @@ include::modules/comm-attributes.adoc[]
 
 = Workflow
 
-These patterns are designed to be composed of multiple components, and for those components to be used in gitops
-workflows by consumers and contributors.  To use the first pattern as an example, we maintain the link:/industrial-edge[Industrial Edge] pattern, which uses a https://github.com/validatedpatterns/industrial-edge[repo] with pattern-specific logic and configuration as well as a https://github.com/validatedpatterns/common[common repo] which has elements common to multiple patterns.  The common repository is included in each pattern repository as a subtree.
+Patterns are designed to be composed of multiple components, that consumers and contributors can integrate into GitOps workflows. For instance, the link:/industrial-edge[Industrial Edge]  pattern comprises a https://github.com/validatedpatterns/industrial-edge[repository] with pattern-specific logic and configuration, alongside a https://github.com/validatedpatterns/common[common repository] containing elements common to various patterns. This common repository is included as a subtree within each pattern repository, streamlining consistency and reusability across different patterns.
+
 
 [id="consuming-a-pattern"]
 == Consuming a pattern
 
-. Fork the pattern repository on GitHub to your workspace (GitHub user or organization). It is necessary to fork because your fork will be updated as part of the GitOps and DevOps processes, and the main branch (by default) will be used in the automated workflows.
-. Clone the forked copy
+. Fork the pattern repository on GitHub to your workspace (GitHub user or organization). Forking is necessary because your fork updates as part of the GitOps and DevOps processes, and the main branch (by default) is used in the automated workflows.
+. Clone the forked repository:
 +
 `git clone git@github.com:<your-workspace>/industrial-edge.git`
 
-. Create a local copy of the Helm values file that can safely include credentials
-
-DO NOT COMMIT THIS FILE
-  You do not want to push personal credentials to GitHub.
-
+. Create a local copy of the Helm values file to safely include credentials.
++
+[NOTE]
+====
+Do not commit this file as it may contain sensitive credentials.
+====
++
 [source,terminal]
 ----
 $ cp values-secret.yaml.template ~/values-secret.yaml
 $ vi ~/values-secret.yaml
 ----
 
-. Customize the deployment for your cluster
+. Customize the deployment for your cluster.
```

Comment:
Can we explain what we mean be customize and separate out the commands into separate commands please.